[FW][FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/odoo-argentina#982

rov-adhoc committed:
[FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings

X-original-commit: 8d27bcc445c087e8906aa8e2e6c5d73c1fb413c1

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/983).

fw-bot-adhoc commented:
@rov-adhoc @maq-adhoc cherrypicking of pull request ingadhoc/odoo-argentina#982 failed.

stdout:
```
Auto-merging l10n_ar_tax/__manifest__.py
CONFLICT (content): Merge conflict in l10n_ar_tax/__manifest__.py
CONFLICT (modify/delete): l10n_ar_account_withholding/views/account_payment_view.xml deleted in HEAD and modified in 4e1d8f7e ([FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings). Version 4e1d8f7e ([FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings) of l10n_ar_account_withholding/views/account_payment_view.xml left in tree.
CONFLICT (modify/delete): l10n_ar_account_withholding/models/account_payment.py deleted in HEAD and modified in 4e1d8f7e ([FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings). Version 4e1d8f7e ([FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings) of l10n_ar_account_withholding/models/account_payment.py left in tree.

```

stderr:
```
19:12:41.262320 git.c:444               trace: built-in: git cherry-pick 4e1d8f7e16ea1417dc6f636fd42283697705f56d
error: could not apply 4e1d8f7e... [FIX] l10n_ar_account_withholding: Fix in payment view and recomputation of withholdings
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
